public/partials/supplies_card.html

```diff
@@ -0,0 +1,73 @@
+
+<md-card-content class="{{newCommodity ? 'dark':'white'}}">
+    <div layout="row">
+        <h3 class="{{newCommodity ? 'dark':'white'}}">Supplies</h3>
+        <div flex class="md-actions edge" layout="row" layout-align="end center">
+            <md-button ng-click="newCommodityItem()" class="md-fab edge" aria-label="Add New Supply">
+                <md-icon class="edge" md-svg-icon="/images/ic_add_24px.svg"></md-icon>
+            </md-button>
+        </div>
+
+    </div>
+
+    <div ng-include="newCommodity ? '/partials/commodity_new.html':null">
+
+    </div>
+    <div ng-include="editCommodity ? '/partials/commodity_edit.html':null">
+
+    </div>
+
+    <style>
```

Comment:
Remove styling from the HTML template and if necessary place it in the application style sheet with a specific class that doesn't affect everything else in the application.